ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -175,17 +175,19 @@ def create_policy(self, brain_parameters: BrainParameters) -> TFPolicy:
         return self.trainer.create_policy(brain_parameters)
 
     def add_policy(self, name_behavior_id: str, policy: TFPolicy) -> None:
-        # for saving/swapping snapshots
```

Comment:
Can you add a proper docstring to this method, it seems important

Reply:
@andrewcoh Should I just use the same docstring from the PPO trainer for the `add_policy` in the ghost trainer?

Reply:
@ervteng That works

Reply:
Added docstring for this and PPO